openfe/analysis/plotting.py

```diff
@@ -5,7 +5,7 @@
 from matplotlib.axes import Axes
 import numpy as np
 import numpy.typing as npt
-from openff.units import unit
+from openff.units import unit, Quantity
```

Comment:
```suggestion
from openff.units import Quantity
```
might not need this any more, here and in some other files